src/js/grapl-cdk/lib/dgraph.ts

```diff
@@ -0,0 +1,195 @@
+import * as cdk from "@aws-cdk/core";
+import * as ec2 from "@aws-cdk/aws-ec2";
+import * as ecs from "@aws-cdk/aws-ecs";
+import * as servicediscovery from "@aws-cdk/aws-servicediscovery";
+
+class Zero extends cdk.Construct {
+    readonly name: string;
+
+    constructor(
+        scope: cdk.Construct,
+        graph_name: string,
+        id: string,
+        cluster: ecs.Cluster,
+        peer: string,
+        idx: number
+    ) {
+        super(scope, graph_name + id);
+
+        const zeroTask = new ecs.Ec2TaskDefinition(
+            this, 
+            id,
+            {
+                networkMode: ecs.NetworkMode.AWS_VPC,
+            }
+        );
+
+        const command = ["dgraph", "zero", `--my=${id}.${graph_name}.grapl:5080`,
+            "--replicas=3",
+            `--idx=${idx}`,
+            "--alsologtostderr"];
+
+        if (peer) {
+            command.push(`--peer=${peer}.${graph_name}.grapl:5080`);
+        }
+
+        const logDriver = new ecs.AwsLogDriver({
+            streamPrefix: `ecs${graph_name + id}`,
+        });
+
+        zeroTask.addContainer(id + 'Container', {
+            // --my is our own hostname (graph + id)
+            // --peer is the other dgraph zero hostname
+            image: ecs.ContainerImage.fromRegistry("dgraph/dgraph"),
```

Comment:
Can we pin to a version of dgraph? This is what we use locally:
`dgraph/standalone:v1.2.2`